images/bootstrap/prunedocker/main.go

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+   Implements cleaning up unwanted docker data root.
+   This is similar to `docker system prune` but tuned to our needs.
+   TODO(bentheelder): rename this. prune-docker <-> docker prune is a bit lame.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"sort"
+	"strings"
+	"time"
+)
+
+var (
+	// images newer than this will be deleted
+	deleteYoungerThanDuration = time.Hour * 24 * 7
```

Comment:
This should be tuned in the future, but this should be a reasonable starting point.